### PR TITLE
22.11 fb grid panel hotfixes

### PR DESCRIPTION
**Pull request by LeviCameron1**
#### Rationale
Hot-fixes for the views of pregnancies and master treatments reports

#### Changes
Changed default view for pregnancies report
Added status and end date columns for master treatments report

### CODE REVIEW

File: WNPRC_EHR/resources/web/wnprc_ehr/reports/PregnancyReport.js
Comment (dnicolalde) [inline WNPRC_EHR/resources/web/wnprc_ehr/reports/PregnancyReport.js:1]: would this apply to everywhere this grid is used?

Comment (dnicolalde) [reply WNPRC_EHR/resources/web/wnprc_ehr/reports/PregnancyReport.js:1]: @LeviCameron1 here is my comment

**Review by dnicolalde — COMMENTED** [review]
looks good

File: WNPRC_EHR/resources/web/wnprc_ehr/reports/PregnancyReport.js
Comment (LeviCameron1) [reply WNPRC_EHR/resources/web/wnprc_ehr/reports/PregnancyReport.js:1]: No only the report in animal history

**Review by dnicolalde — APPROVED** [review]
thank you

**Pull request merged**